run_all_experiments.py

```diff
@@ -128,6 +129,65 @@ def get_experiment_configs(
   return [(config, args) for config in experiment_configs]
 
 
+def has_cache_build_script(project):
+  cached_build_script = os.path.join('fuzzer_build_script', project)
+  if os.path.isfile(cached_build_script):
+    return True
+  return False
+
+
+def prepare_image_cache(project):
+  # Only create a cached image if we have a post-build build script
+  if not has_cache_build_script(project):
+    return False
+
+  cached_container_name = f'cached_container_{project}'
+  adjusted_env = os.environ | {
+      'OSS_FUZZ_SAVE_CONTAINERS_NAME': cached_container_name
+  }
+
+  # Create cached image by building using OSS-Fuzz with set variable
+  command = ['python3', 'infra/helper.py', 'build_fuzzers', project]
```

Comment:
I reckon we can build the cache image of all projects every day (if we have their `fuzzer_build_script` in your solution, or address the TODOs in mine).
OSS-Fuzz is already building all original projects anyway.

We run multiple experiments every day, no need to rebuild the cache images in every experiment.

Reply:
I'm fine with that, I say we land it now as is and then do the migration of OSS-Fuzz build images/images build from cloud in the next steps. It's working well locally and I think that's a nice stage